.circleci/config.yml

```diff
@@ -197,6 +197,7 @@ workflows:
           requires:
             - prep-deps
       - test-e2e-chrome-webpack:
+          <<: *develop_master_rc_only
```

Comment:
nit: maybe we can also apply the same filter to the `prep-build-test-webpack` job?

Reply:
Good suggestion, I had the same thought as well. I wasn't sure though, as having the build might catch build errors that we would otherwise miss.

Reply:
good point, yeah we won't win much from that change, compared to the information we can loose, so happy as it is :+1: